src/Constants.py

```diff
@@ -1,5 +1,11 @@
 from enum import Enum
 
+# Persistent data directories
+BASE_DIR = "../trd-data"
+CONFIG_DIR = "/configurations"
+SIMULATIONS_DIR = "/simulations"
```

Comment:
Please remove the `/`. When we make `os.path.join(args.reports_base, SIMULATIONS_DIR)` later on the function `os.path.join` thinks you give it two absolute paths from which it takes second one. The diskfull regression test tries to create a /simulations folder in the root for which it then throws a Permission Denied error.